<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NonUserInterface_UI</class>
 <widget class="QWidget" name="NonUserInterface_UI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1420</width>
    <height>780</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="LeftButtons_NonUserUI_LBL">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>151</width>
     <height>801</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../../Desktop/CS1D/Project Application/Non-User Interface/LeftSidePNGNonuserInterface/LeftSideNonuserInterface.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="NonUserInterface_UI_Map_BTN">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>440</y>
     <width>151</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="NonUserInterface_UI_Weather_BTN">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>590</y>
     <width>151</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="NonUserInterface_UI_MLBNews_BTN">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>280</y>
     <width>151</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="NonUserInterface_UI_Search_BTN">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>151</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="NonUserInterface_Hotbar_UI_LBL">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>0</y>
     <width>1291</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../../Desktop/CS1D/Project Application/Non-User Interface/HotbarPNGNonUserInterface/HotbarNonuserInterface_Home.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="NonUserInterface_MidboxContent_UI_LBL">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>90</y>
     <width>1301</width>
     <height>691</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../../Desktop/CS1D/Project Application/Non-User Interface/MidBoxContentNonuserInterface/MidPartNonuserInterface.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="NonUserInterface_Home_BTN">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="NonUserInterface_About_Us_BTN">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>60</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="NonUserInterface_Services_BTN">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>60</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="NonUserInterface_Tutorial_BTN">
   <property name="geometry">
    <rect>
     <x>502</x>
     <y>60</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="NonUserInterface_Login_BTN">
   <property name="geometry">
    <rect>
     <x>1232</x>
     <y>60</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="NonUserInterface_Register_BTN">
   <property name="geometry">
    <rect>
     <x>1312</x>
     <y>68</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWebView" name="NonUserInterface_WebView">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>90</y>
     <width>1271</width>
     <height>691</height>
    </rect>
   </property>
   <property name="url">
    <url>
     <string>about:blank</string>
    </url>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
